src/components/HomepageFeatures/index.tsx

```diff
@@ -82,6 +82,25 @@ const FeatureList: FeatureItem[] = [
     ),
     to: [{ link: '/ads', text: 'Get Started' }],
   },
+  {
+    title: 'Dolby OptiView Ad Engine',
+    Image: (props) => (
+      <ThemedImage
+        {...props}
+        sources={{
+          light: useBaseUrl('/img/ads-black-wordmark.svg'),
+          dark: useBaseUrl('/img/ads-white-wordmark.svg'),
+        }}
```

Comment:
It looks kinda weird to have two cards with the same logo side-by-side.

![Screenshot](https://github.com/user-attachments/assets/63528067-e52e-4169-a5fe-bb60b8c30585)

Should we perhaps put Ads and Ad Engine on the same card, like we do for Streaming?
![Screenshot](https://github.com/user-attachments/assets/53d16e27-3843-47f3-b53c-06575a6be2a2)